samples/RNWinRTTestApp/windows/RNWinRTTestApp.sln

```diff
@@ -1,52 +1,13 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.0.32014.148
+# Visual Studio Version 16

```

Comment:
Should VS 2022 be the default?

Reply:
I had to re-generate the solution to get the ExperimentalNuget stuff working and just kept whatever it generated. From what I've experienced, there's been no issues with running the sample using VS 2022

Reply:
we don't currently work with 2022 out of the box (e.g. 2022 has vc143 as the default toolchain, and RNW uses vc142, so it requires you installing additional non-default toolchain in VS). Once you do that, it should mostly just work (minus the compiler getting stricter in 2022 and uncovering issues in our code, which we should fix)